python/ray/monitor.py

```diff
@@ -219,6 +216,16 @@ def _run(self):
             # round of messages.
             time.sleep(AUTOSCALER_UPDATE_INTERVAL_S)
 
+    def update_event_summary(self):
+        # Periodically emit event summaries.
+        avail_resources = self.load_metrics.resources_avail_summary()
+        if avail_resources != self.last_avail_resources:
+            self.event_summarizer.add(
+                "Resized to {}.",
+                quantity=avail_resources,
+                aggregate=lambda a, b: b)
```

Comment:
Can you please use more meaningful names than a, b?

Reply:
Also, can you cast the resource to INT? it seems to me that the ".0" in 172.0 CPUS is redundant.

Reply:
> Can you please use more meaningful names than a, b?

I think it's pretty standard usage to use generic variable names in lambdas, and I don't giving these longer names would help explain what's going on.

Definitely agree though with the above comment to elaborate on what's being updated.

Reply:
Fixed to {old, new}.